contracts/README.md

```diff
@@ -76,9 +76,32 @@ Delete the smart contract artifacts, the coverage reports and the Hardhat cache:
 $ yarn clean
 ```
 
+### Deployment
+
+Deploy the contracts to a local hardhat fork by running:
+
+```sh
+$ yarn deploy:poc --network hardhat
+```
+
+You will now have a timestamped file in `./deploy-history` containing the log output of this deployment prefixed with `hardhat-`. You'll also see the `hardhat-latest.json` file. The `-latest.json` file for each network is overwritten each time the command runs. Hardhat deployments are ignored and should not be committed to the repo.
+
+To deploy to a "real" network like mainnet or a testnet, replace the network name, as defined in `hardhat.config.ts`.
+
+```sh
+$ yarn deploy:poc --network mainnet
```

Comment:
Mainnet isn't configured in the hardhat config so this currently won't work—should we add the `mainnet` network in this PR?

Reply:
Captured in https://github.com/dcgtc/dgrants/issues/186